mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -1859,11 +1863,11 @@ LogicalResult SimdOp::verify() {
 
 void DistributeOp::build(OpBuilder &builder, OperationState &state,
                          const DistributeOperands &clauses) {
-  // TODO Store clauses in op: privateVars, privateSyms.
-  DistributeOp::build(
-      builder, state, clauses.allocateVars, clauses.allocatorVars,
-      clauses.distScheduleStatic, clauses.distScheduleChunkSize, clauses.order,
-      clauses.orderMod, /*private_vars=*/{}, /*private_syms=*/nullptr);
+  DistributeOp::build(builder, state, clauses.allocateVars,
+                      clauses.allocatorVars, clauses.distScheduleStatic,
+                      clauses.distScheduleChunkSize, clauses.order,
+                      clauses.orderMod, clauses.privateVars,
+                      getPrivateSymsAttrIfPresent(builder, clauses));
```

Comment:
I think calling `makeArrayAttr(ctx, clauses.privateSyms)`, as it's done for other builders for operations including the `private` clause, would be better than adding another helper function.

Reply:
Of course, I did not notice this one, thanks!